panel/models/ipywidget.ts

```diff
@@ -59,13 +54,26 @@ export class IPyWidgetView extends HTMLBoxView {
 
   render(): void {
     super.render()
-    if (this.ipyview != null) {
-      this.shadow_el.appendChild(this.ipyview.el)
-      this.ipyview.trigger('displayed', this.ipyview)
-      for (const child of this.ipychildren)
-        child.trigger('displayed', child)
-      this.invalidate_layout()
-    }
+    const {spec, state} = this.model.bundle
+    this.manager.set_state(state).then(async (models: any) => {
+      const model = models.find((item: any) => item.model_id == spec.model_id)
+      if (model == null)
+	return
+
+      this.manager.create_view(model, {el: this.el}).then(async (view: any) => {
```

Comment:
Given the enclosing function is `async`, you can simply use `const view = await this.manager.create_view ...`.

Reply:
Indeed.